app/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from pytest import fixture
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+
+
+@fixture
+def user_factory(db):
```

Comment:
The name is a bit ambiguous: it's about the user that's used to authenticate on the API, but that isn't exactly clear. It could be understood as the users in the db, the ones that we push to cognito.

Reply:
Renamed to django_user_factory